src/forms/Select.jsx

```diff
@@ -74,14 +78,16 @@ export const SelectInput = (props: Props) => {
 				</label>
 			)}
 			{helperText && <div className={classNames.helperText}>{helperText}</div>}
-			<select
-				name={name}
-				id={id || name}
-				required={Boolean(required)}
-				className={classNames.field}
-				{...other}
-			/>
-			<Icon className="select-customArrow" shape="chevron-down" size="xs" />
+			<div className={cx('selectWrapper', selectClassName)}>
```

Comment:
Didn't Dr. Mike say we should be using cx only when we really need it, but that for something like this we could just do `selectWrapper ${selectClassName}`?

Reply:
still personally prefer the other way, but updated :)

Reply:
This case, where you're combining a string with a variable, is sort of borderline on which implementation is 'better', so I personally wouldn't call it out either way. String templates in JS aren't particularly elegant, but I don't think either approach is especially awkward, so unless you feel strongly about it, don't feel compelled to call it out in the future :).